assessments/KRAssessment/earthmover.yaml

```diff
@@ -0,0 +1,311 @@
+        join_type: left
+      - operation: join
+        sources:
+          - $transformations.performance_level_descriptors
+        join_type: inner
+        left_key: assessmentIdentifier
+        right_key: assessmentIdentifier  
+      - operation: add_columns
+        columns:
+          PerformanceLevel_SFScore: "{%raw%}{% if SFScore >= minimumScore and SFScore <= maximumScore %}{{ codeValue }}{% endif %}{%endraw%}"
+          PerformanceLevel_LLScore: "{%raw%}{% if LLScore >= minimumScore and LLScore <= maximumScore %}{{ codeValue }}{% endif %}{%endraw%}"
+          PerformanceLevel_MAScore: "{%raw%}{% if MAScore >= minimumScore and MAScore <= maximumScore %}{{ codeValue }}{% endif %}{%endraw%}"
+          PerformanceLevel_PDScore: "{%raw%}{% if PDScore >= minimumScore and PDScore <= maximumScore %}{{ codeValue }}{% endif %}{%endraw%}"
+      - operation: filter_rows
+        query: "(PerformanceLevel_SFScore != 'Unknown' ) or (PerformanceLevel_LLScore != 'Unknown') or (PerformanceLevel_MAScore != 'Unknown') or (PerformanceLevel_PDScore != 'Unknown' )"
+
+        behavior: include
+      - operation: group_by
+        group_by_columns:
+          - studentAssessmentIdentifier
+          - assessmentIdentifier
+          - studentUniqueId
+          - administrationDate
+          - SchoolYear
+          - namespace
+          - whenAssessedGradeLevelDescriptor
+          - OverallScore
+          - PerformanceLevelDescriptor
+          - SFScore
+          - LLScore
+          - MAScore
+          - PDScore
+          - schoolId
+        create_columns: 
+          PerformanceLevel_SFScore: agg(PerformanceLevel_SFScore,)
+          PerformanceLevel_LLScore: agg(PerformanceLevel_LLScore,)
+          PerformanceLevel_MAScore: agg(PerformanceLevel_MAScore,)
+          PerformanceLevel_PDScore: agg(PerformanceLevel_PDScore,)
+               
+
+destinations:
+  assessments:
+    source: $transformations.prepare_assessments
+    template: ${BUNDLE_DIR}/templates/assessments.jsont
+    extension: jsonl
+  objectiveAssessments:
+    source: $transformations.prepare_objective_assessments
+    template: ${BUNDLE_DIR}/templates/objectiveAssessments.jsont
+    extension: jsonl
+  studentAssessments:
+    source: $transformations.prepare_student_assessments
+    template: ${BUNDLE_DIR}/templates/studentAssessments.jsont
+    extension: jsonl
+  studentAssessmentEducationOrganizationAssociations:
+    source: $transformations.prepare_student_assessments
+    template: ${BUNDLE_DIR}/templates/studentAssessmentEducationOrganizationAssociations.jsont
+    extension: jsonl
+  assessmentReportingMethodDescriptors:
+    source: $sources.assessmentReportingMethodDescriptors
+    template: ${BUNDLE_DIR}/templates/descriptors.jsont
+    extension: jsonl
+  performanceLevelDescriptors:
+    source: $sources.performanceLevelDescriptors
+    template: ${BUNDLE_DIR}/templates/descriptors.jsont
+    extension: jsonl
+  academicSubjectDescriptors:
```

Comment:
Typically we don't send our own subjects, we usually try to align to values that already exist in the ODS